dev-cpp/websocketpp/patches/websocketpp-0.8.1.patchset

```diff
@@ -0,0 +1,139 @@
+From 244a4d4331d714aa3e06916edd6599a6fe7cbe01 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 14 Sep 2019 10:27:13 +0200
+Subject: Set correct path for $includeDir
+
+
+diff --git a/cmake/CMakeHelpers.cmake b/cmake/CMakeHelpers.cmake
+index 1478f4b..f603632 100644
+--- a/cmake/CMakeHelpers.cmake
++++ b/cmake/CMakeHelpers.cmake
+@@ -80,7 +80,7 @@ macro (final_target)
+     endif ()
+ 
+     install (DIRECTORY ${CMAKE_SOURCE_DIR}/${TARGET_NAME}
+-             DESTINATION include/
++             DESTINATION ${INSTALL_INCLUDE_DIR}/
+              FILES_MATCHING PATTERN "*.hpp*")
+ endmacro ()
+ 
+-- 
+2.23.0
+
+
+From a76e06c2aff7391c7dc6beaae798436e33194526 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 14 Sep 2019 13:52:28 +0200
+Subject: Disable test test_transport
+
+
+diff --git a/test/transport/CMakeLists.txt b/test/transport/CMakeLists.txt
+index 6b44d98..db0922d 100644
+--- a/test/transport/CMakeLists.txt
++++ b/test/transport/CMakeLists.txt
+@@ -1,15 +1,5 @@
+ if (OPENSSL_FOUND)
+ 
+-# Test transport integration
+-file (GLOB SOURCE integration.cpp)
+-
+-init_target (test_transport)
+-build_test (${TARGET_NAME} ${SOURCE})
+-link_boost ()
+-link_openssl()
+-final_target ()
+-set_target_properties(${TARGET_NAME} PROPERTIES FOLDER "test")
+-
+ # Test transport asio timers
+ file (GLOB SOURCE asio/timers.cpp)
+ 
+-- 
+2.23.0
+
+
+From 3ee67896d845b6f4e763e895f90cada10d9314f5 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 14 Sep 2019 14:38:58 +0200
+Subject: Update version number in CMakeList.txt (import merged PR)
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2786aba..2d13117 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -24,7 +24,7 @@ endif ()
+ ############ Project name and version
+ set (WEBSOCKETPP_MAJOR_VERSION 0)
+ set (WEBSOCKETPP_MINOR_VERSION 8)
+-set (WEBSOCKETPP_PATCH_VERSION 0)
++set (WEBSOCKETPP_PATCH_VERSION 1)
+ set (WEBSOCKETPP_VERSION ${WEBSOCKETPP_MAJOR_VERSION}.${WEBSOCKETPP_MINOR_VERSION}.${WEBSOCKETPP_PATCH_VERSION})
+ 
+ if(POLICY CMP0048)
+-- 
+2.23.0
+
+
+From cb480d1403bf38d0853224d4d132e840e23d703e Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 28 Sep 2019 08:08:30 +0200
+Subject: Add Haiku changes
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2d13117..08934ce 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -3,6 +3,7 @@
+ # Minimum cmake requirement. We should require a quite recent
+ # cmake for the dependency find macros etc. to be up to date.
+ cmake_minimum_required (VERSION 2.8.8)
++set(CMAKE_TRY_COMPILE_TARGET_TYPE "STATIC_LIBRARY")
```

Comment:
Not sure this is really needed (as mentioned on the original PR), I don't see any reference on the github site for websocketpp ...